docs/_blog/spending_policy_demo.md

```diff
@@ -31,11 +31,11 @@ In a real-world wallet a longer relative time-lock would probably be used, but w
 ### Step 0: Install a recent version `bdk-cli`
 
 ```bash
-cargo install bdk-cli --features repl,electrum,esplora
+cargo install bdk-cli
```

Comment:
We do need at least one blockchain feature added, otherwise we can use any online commands like `sync`.
```suggestion
cargo install bdk-cli --features electrum
```

Reply:
Oh sorry I thought that the `electrum` feature was added by default?

Reply:
Ah ya the original issue is based on older version of bdk-cli where the `electrum` feature was enabled by default. But when we added the ability to have different blockchains like esplora or rpc we had to remove it as a default. I hope in the future we'll be able to have all features enabled without conflicting, then we can make electrum added by default again. See https://github.com/bitcoindevkit/bdk-cli/issues/75.

Reply:
I realize the default electrum feature was actually removed by #75. Re-added.